jekyll/_layouts/classic-docs.html

```diff
@@ -29,7 +29,7 @@
       {% include_cached sidebar.html lang=lang sidenav=sidenav current_url=current_url %}
     <div class="article-toc-footer">
       <div class="article-toc">
-        <div class="article-container docs" id="main">
+        <div class="article-container docs {{page.title | slugify}}" id="main">
```

Comment:
This was done so that we can easily scope scss to specific pages without worrying about breaking other things.